noxfile.py

```diff
@@ -147,6 +128,25 @@ def run_pytest(
     )
 
 
+@nox.session(python=_SUPPORTED_PYTHONS)
+def get_unit_tests(session: nox.Session) -> None:
```

Comment:
do we need all these function in nox? should we just pass the relevant arguments to a single nox session?

Reply:
hmm the original thought was to keep the way the circleci file was originally written which was to be general, and use various nox sessions to handle finding the files.

but after messing around and looking at the changes I think this approach is best.
We will still have to duplicate some copying of declaring the tests path in the .circleci config. But this should be clearer when looking at circleci what tests will be ran/ignored